module/Calibrator.cpp

```diff
@@ -57,13 +57,13 @@ void Calibrator::measureTargetBrightness()
   char buf[BUF_SIZE];  // log用にメッセージを一時保持する領域
   Logger logger;
 
-  // ライン上で中央ボタンを押して、黒と白の中間色の輝度を取得する
-  logger.log("Press the Center Button on the Line");
-  // 中央ボタンが押されるまで待機
-  while(!measurer.getEnterButton()) {
+  // ライン上で右ボタンを押して、黒と白の中間色の輝度を取得する
+  logger.log("Press the Right Button on the Line");
+  // 右ボタンが押されるまで待機
+  while(!Measurer::getRightButton()) {
     timer.sleep();  // 10ミリ秒スリープ
   }
```

Comment:
外れ値を引いたときに手軽に再取得できた方がいいかも。（実装はなんでもいい）

```suggestion
  targetBrightbess = -1;
  // 左ボタンで輝度を取得し、右ボタンで待機状態に入る
  while(targetBrightness < 0 || !Measurer::getRightButton()){
    // 左ボタンが押されるまで待機
    while(!Measurer::getLeftButton()){
      timer.sleep(10);
    }
    // 輝度取得
    targetBrightness = Measurer::getBrightness();
    snprintf(buf, BUF_SIZE, ">> Target Brightness Value is %d", targetBrightness);
    logger.log(buf);
    timer.sleep(10);  // 10ミリ秒スリープ
  }
```

Reply:
Ctrl＋Zで抜けると、高確率でkillできないプロセスが残って現状再起動しないと消せないから、できるだけ中断したくない